scripts/pi-hole/php/teleporter.php

```diff
@@ -148,7 +148,7 @@ function getWildcardListContent() {
 else
 {
 	$filename = "pi-hole-teleporter_".date("Y-m-d_h-i-s").".zip";
-	$archive_file_name = "/var/www/html/".$filename;
+	$archive_file_name = "/etc/pihole/".$filename;
```

Comment:
Looks like a fresh install has these permissions:
```
root@ubuntu-512mb-nyc2-01:~# ll /etc/pihole/
total 10336
drwxr-xr-x  2 root root    4096 Apr 28 20:27 ./
[...]
```
So either the permissions need to be changed, or we can use a different directory, like `/var/www`.